src/psyclone/psyGen.py

```diff
@@ -4465,6 +4663,227 @@ def _number_handler(self, node, parent):
         return Literal(node.items[0], parent=parent)
 
 
+class Symbol(object):
+    '''
+    Symbol item for the Symbol Table. It contains information about: the name,
+    the datatype, the shape (in row-major order) and the symbol
+    access characterisctics. The symbol access can be:
+        - local: Variable that just exist in the kernel scope.
```

Comment:
Great documentation. Please put the possible symbol access values in quotes to show that they are strings, i.e. "local": Variable that...